pages/thc/[processRef]/review.tsx

```diff
@@ -121,25 +129,18 @@ const ReviewPage: NextPage = () => {
       pausable
     >
       <React.Fragment>
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+        />
         <SummaryList
           className="govuk-summary-list--no-border mat-tenancy-summary"
           rows={[
-            {
-              key: "Address",
-              value: address.result ? address.result.join(", ") : "Loading...",
-            },
-            {
-              key: "Tenants",
-              value:
-                allTenantNames.length > 0
-                  ? allTenantNames.join(", ")
-                  : "Loading...",
-            },
-            {
-              key: "Tenure type",
-              value: tenureType.result ? tenureType.result : "Loading...",
-            },
-            { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+            { key: "Outside Property:", value: outsidePropertyImageThumbnails },
```

Comment:
```suggestion
            { key: "Outside Property", value: outsidePropertyImageThumbnails },
```

I think (without looking).

Reply:
I actually took this off the balsamiq design, but can get rid of it if you think it'd look better without?

Reply:
I think it's more consistent?